gtsam/hybrid/tests/testGaussianMixtureFactor.cpp

```diff
@@ -504,87 +538,110 @@ TEST(GaussianMixtureFactor, TwoStateModel2) {
   using namespace test_two_state_estimation;
 
   double mu0 = 1.0, mu1 = 3.0;
-  double sigma0 = 6.0, sigma1 = 4.0;
-  auto model0 = noiseModel::Isotropic::Sigma(1, sigma0);
-  auto model1 = noiseModel::Isotropic::Sigma(1, sigma1);
-
-  DiscreteKey m1(M(1), 2);
-  Key z0 = Z(0), z1 = Z(1);
+  double sigma0 = 0.5, sigma1 = 2.0;
```

Comment:
Again not sure why these values were changed. I would expect that we would still keep sigma0 > sigma1.

Reply:
Same.

Reply:
Tell me why, though? I might have forgotten why that was important...

Reply:
The idea was to model the foot contact, with sigma0 being swing and sigma1 being stance hence a tighter noise model. If this test case passed, then the same principle would work for our contact estimation.